### ISSUE TITLE
Add CS resources on different topics

### ISSUE THREAD

**Issue by vicky002** [vicky002/AlgoWiki#65]
This repository contains links to different CS guides, articles, and resources on the web. If you think some cool tool is missing in this repo please make a Pull request. 🎉  

Thanks for being awesome. Happy Hacktoberfest.

Edit: 26th Sept 2022 — You can still contribute on many topics and raise PRs. Contributions are welcome, always 🚀

**Comment by digilou** [comment]
Would you like to add Ruby and Rails to your list? I have a few suggestions...

**Comment by chanakya-vc** [comment]
Hey @vicky002 , Would you like to add few resources to basic intro to device drivers and system programming in your Linux section? I see that you have the basics already covered. Nice work by the way!

**Comment by vicky002** [comment]
@digilou and @chanakya-vc sure go-ahead. 

Make sure it's in the proper folder and well formatted. You can add any resources related to any topic in Computer Science. 

So, Looking forward to it. 👍 🌮

**Comment by vkartik97** [comment]
Can I add Tricks on running test cases for competitive programming, on sublime?

**Comment by vkartik97** [comment]
@vicky002 
Can I add resources to `Free-Video-Courses` and add `Latex` to resources?

**Comment by vkartik97** [comment]
@vicky002 
Can I add `Latex` to resources?

**Comment by vkartik97** [comment]
@vicky002 
Can I add `Latex` to Resource List?

**Comment by vicky002** [comment]
yes you can @vkartik97

**Comment by ArleneAndrews** [comment]
I have one resource to add for free-video-courses -- let me verify the link is correct and functional.

**Comment by JuanLoLuna** [comment]
Hi @vicky002 Can I add some CSS frameworks to the HTML_CSS resources?

**Comment by vicky002** [comment]
Hey @JuanLoLuna , yes you can.

**Comment by RoshanSalian** [comment]
I have ppt of algorithms of IITB. Can I add them?

**Comment by vicky002** [comment]
@RoshanSalian a link would be better as PPT will take lots of space, and time in cloning as well. 

Upload on drive or Dropbox somewhere.

**Comment by malevolentninja** [comment]
Regarding JavaScript frameworks should I add them to the JavaScript readme or create a separate list of links etc?

**Comment by vicky002** [comment]
@malevolentninja l think JavaScript Readme would be better.

**Comment by prashi23** [comment]
Hi @vicky002 Can I add Computer networking resources?

**Comment by vicky002** [comment]
Hi @prashi23 yes sure 💯

**Comment by vk-mittal14** [comment]
Can I add some Machine Learning & Deep Learning related resources

**Comment by spuru9** [comment]
@vicky002 I have added React Resources.

**Comment by spuru9** [comment]
@vicky002 Can I add resources on AWS (tutorials and courses)? If yes, in which section?

**Comment by AryanNayak** [comment]
Hey, is this issue open I would like to contribute?

**Comment by ndhoang123** [comment]
Hi @vicky002, I think Database is essential and fundamental knowledge in CS. Hopefully, in the future, you guys will create a folder for the topic.

**Comment by 21pravi** [comment]
Hello there! I am new to the open source world and would like to contribute to some real-world projects. If this issue is not resolved yet, can I be assigned to it?

**Comment by vicky002** [comment]
@21pravi yes this is still open to contribute. Feel free to contribute on any topics and raise PR.

**Comment by BishalBiswal** [comment]
Hi, I'm new to open source contributions and I would like to contribute to this repo. If this issue is not resolved yet, can I be assigned to it?

**Comment by BhumikaJena** [comment]
Hey, Can I add resources on DevOps? If yes, in which section?

**Comment by SyedTathahir** [comment]
Hey can I add resource on networking fundamentals?
